src/components/task/existing-card/index.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import propTypes from "prop-types";
+import dayjs from "dayjs";
+
+import Button from "../../base/button";
+import Image from "../../base/image";
+import Styles from "./index.module.scss";
+import TextArea from "../../base/text-area";
+
+function TaskCard({
+  title,
+  createdAt,
+  isCompleted,
+  completedAt,
+  isTaskOnEditMode,
+}) {
+  const [isTextAreaVisible, setIsTextAreaVisible] = useState(isTaskOnEditMode);
+
+  const calculateDateDifference = (startDate, endDate = dayjs()) =>
+    endDate.diff(startDate, "day");
+  const formatDate = (date) => dayjs(date, "YYYY-MM-DD").format("YYYY-MM-DD");
+
+  return (
+    <div className={Styles.card}>
+      {isTextAreaVisible ? <TextArea noOfRows={5} /> : <h3>{title}</h3>}
+
+      <p className={Styles.date}>created At: {formatDate(createdAt)}</p>
+
+      <div className="flex justify-between">
+        <div className="flex items-center">
+          <Button className={`${Styles.button} ${Styles.doneBtn}`}>
+            <Image src="check.png" alt="check" />
```

Comment:
Can we be more specific about the alt, like what is the Image actually representing here?

Reply:
we could name this as "Complete Button" and import the const.
```suggestion
            <Image src="check.png" alt={COMPLETE_BUTTON} />
```